### DISCUSSION TITLE
Request for a full cypress example

### DISCUSSION THREAD

**Discussion by pastrom** [zendesk/laika#60] [Q&A]
Awesome work on this tool - its just what I've been looking for. 

Although, I haven't managed to set up a working Cypress for the project Im working on. I've been trying to mock both gql queries (http) and subscriptions (ws) made by my Angular components using Apollo. To confirm the basic understanding;

- In the Cypress example you have posted (https://zendesk.github.io/laika/docs/usage-in-cypress/) you refer to a clientName in the intercept matcher object. Is it correct that one have to create this laika client in the Angular component where the Apollo client is created? 
(as you have done on the 'Installation and usage' part of your Medium article https://zendesk.engineering/mock-graphql-operations-with-laika-the-only-apollo-client-mocking-tool-youll-need-c116e1f62128)

**Comment by niieani** [comment]
Thanks @pastrom! Yes, that's correct. You need to use the Laika link when you create the Apollo Client to be able to use Laika in Cypress tests. You can conditionally add the link only when running Cypress (checking for `window.CYPRESS` I believe).
